Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -199,8 +199,13 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
         .take(first: 1)
     )
 
-    self.projectImageUrl = project
-      .map { URL(string: $0.photo.full) }
+    self.projectImageUrl = Signal.merge(
+      configData.mapConst(nil),
+      Signal.combineLatest(
```

Comment:
@mbrandonw this was a weird one, even though `configData` is _definitely_ emitting before `project` (because `project` is `configData.map(first)`, I couldn't seem to get `nil` to emit first without merging in that `combineLatest`?